### DISCUSSION TITLE
Rejected for GitHub Student Pack Application - Need Assistance

### DISCUSSION THREAD

**Discussion by TennousuAthena** [community/community#66087] [GitHub Education]
### Hi everyone,

Hi, I am a student at Northeastern University in China, however, my request was rejected due to the following:
- Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.
- You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document plus a short bio.
- The image you selected does not appear to contain your school name. 

I had already verified our school's email.
I uploaded my School ID card but it does not include the school's English name.
 
I would greatly appreciate any help, guidance, or personal experiences you can share to help me overcome these obstacles and successfully obtain the GitHub Student Pack. Thank you in advance for your assistance!

Best regards,
Athena

**Comment by arafajs** [comment]
1. **Modify your browser settings to allow GitHub to use your location information.** This is necessary because GitHub uses your location to verify that you are a student at a participating school. You can find instructions on how to do this in GitHub's help documentation.
2. **Complete your GitHub user profile with your full name exactly as it appears in your academic affiliation document.** This is also necessary for verification. Make sure to include your full name, your school's name, and your student ID number.
3. **Upload a different image that clearly shows your school name.** You can try uploading a screenshot of your school's website, a letter from your school, or a document that shows your enrollment status.
4. **Contact GitHub support for help.** If you are still having trouble, you can contact GitHub support for help. They may be able to provide additional guidance or escalate your application to a human reviewer.

Here are some additional tips that may help you get your application approved:

* Make sure that the image you upload is clear and legible.
* Make sure that the date on your document is current.
* If you are uploading a document that is not in English, make sure to include an English translation.
* Be patient. The verification process can take a few days.
